### PR TITLE
[WIP] ChatOps document updates

### PR DESCRIPTION
**Pull request by manasdk**

### CODE REVIEW

**Comment by jfryman** [comment]
Picking this up...

File: docs/source/chatops/index.rst [hunk 1]
```diff
@@ -1,8 +1,53 @@
...
+
+Not only does ChatOps reduce the feedback loop of work output, it also empowers others to accomplish complex, self-service tasks that they otherwise would not be able to do. Combining ChatOps and |st2| is an ideal combination, where from Chat users will be able to execute actions and workflows to accelerate the IT delivery pipeline. In the same way, the ChatOps/|st2| combination also enhances user adoption of automation through transparency and consistent execution.
+
+The end result is improved agility and enhanced trust between teams. What’s not to love about this? It’s the reason we as a company are devoted to including it as a core part of our product.
+
+TL;DR
+-----
```
Comment (dzimine) [inline docs/source/chatops/index.rst]: this doesn't seem to work. What are you trying to do? 
image

Comment (DoriftoShoes) [inline docs/source/chatops/index.rst]: Are you talking about the TL;DR?  It is supposed to be like that :)  It stands for "Too Long; Didn't Read".

Comment (dzimine) [inline docs/source/chatops/index.rst]: Lol. buy me an urban slang dict :)

**Pull request merged**